OsmAnd-api/src/net/osmand/aidlapi/map/SetMapLocationParams.java

```diff
@@ -56,6 +66,7 @@ public void writeToBundle(Bundle bundle) {
 		bundle.putDouble("latitude", latitude);
 		bundle.putDouble("longitude", longitude);
 		bundle.putInt("zoom", zoom);
```

Comment:
new fields always should be last parameter

cc @Chumva

Reply:
It is ok to add new fields in Bundle in a different order, but preferably add them at the end for better understanding differences among versions. But in parcelable order of written and read values is important and changes in it will break reading values.

Reply:
Do you think it's better to reorder everything even the parameters order in the constructor, or just the putFloat?

Reply:
`putFloat` and `getFloat` will be enough